### ISSUE TITLE
Small issue we could warn about: binding to content attributes when there's content already

### ISSUE THREAD

**Issue by MichaelRFairhurst** [dart-lang/angular_analyzer_plugin#234]
See this SO post: http://stackoverflow.com/questions/38992712/angular2-dart-component-template-not-loading

basically `<x [innerText]="y">z</x>` should report a conflict between `z` and `y`.

same for `innerHtml`, which I think is used more often than `innerText`. And maybe a few other attributes.

**Comment by zoechi** [comment]
Might cause quite some false warnings.
I'd expect the static content to be the default value

```html
<x [innerText]="y">Content is loading ...</x>
```

**Comment by MichaelRFairhurst** [comment]
Yeah, this is a tricky one. We'd almost want to suggest both things: "Hey, this is blank until change detectors fill it" and "hey, this content gets overwritten!"

**Comment by zoechi** [comment]
I guess using a default value won't work anyway. If `y` is `null` the content will be purged at first change detection run. So, I don't see any use for content in elements with `[innerHTML]` binding.
